### PR TITLE
document upsert sinks

### PR DESCRIPTION
**Pull request by umanwizard**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/5595)

### CODE REVIEW

File: doc/user/content/sql/create-sink.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ _sink&lowbar;name_ | A name for the sink. This name is only used within Material
 _item&lowbar;name_ | The name of the source or view you want to send to the sink.
 **AVRO OCF** _path_ | The absolute path and file name of the Avro Object Container file (OCF) to create and write to. The filename will be modified to let Materialize create a unique file each time Materialize starts, but the file extension will not be modified. You can find more details [here](#avro-ocf-sinks).
 **AS OF** _timestamp&lowbar;expression_ | The logical time to tail from onwards (either a number of milliseconds since the Unix epoch, or a `TIMESTAMP` or `timestamp with time zone`).
+**ENVELOPE DEBEZIUM** | The generated schemas have a [Debezium-style diff envelope](#debezium-envelope-details) to capture changes in the input view or source. This is the default.
```
Comment (JLDLaughlin) [inline doc/user/content/sql/create-sink.md:31]: Does `input view or source` here mean view or source in Materialize?

File: doc/user/content/sql/create-sink.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ _sink&lowbar;name_ | A name for the sink. This name is only used within Material
...
+**ENVELOPE UPSERT** | The sink emits data with upsert semantics: updates and inserts for the given key are expressed as a value, and deletes are expressed as a null value payload in Kafka.
```
Comment (JLDLaughlin) [inline doc/user/content/sql/create-sink.md:32]: Maybe link to the [upsert docs](https://materialize.com/docs/sql/create-source/text-kafka/#upsert-envelope-details) here?

**Review by JLDLaughlin — APPROVED** [review]

File: doc/user/content/sql/create-sink.md (hunk 1 above)
Comment (umanwizard) [reply doc/user/content/sql/create-sink.md:31]: Yes... (btw, I didn't write this; I just moved it from somewhere else)

File: doc/user/content/sql/create-sink.md (hunk 2 above)
Comment (umanwizard) [reply doc/user/content/sql/create-sink.md:32]: Good call, will do

**Pull request merged**